internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -44,6 +45,11 @@ type ContainersConfig struct {
 	Secrets   map[string]string `yaml:"secrets"`
 }
 
+// LogsConfig is the configuration to the ECS logs.
+type LogsConfig struct {
+	LogRetention int `yaml:"logRetention"`
```

Comment:
nit: I might be wrong, but I think you can write `time.Duration` here instead.
This way the constant can be written as:
```go
logRetentionDuration = 30 * 24 * time.Hour 
```

Reply:
I don't think we need this...because the cloudformation just support values all in days and also for now we just want to do it simple and this pattern will be changed in the future. In this PR we want to change the default value from `infinity` to `30 days`.

When we expose this field then we will make it to be sth like `15d`, `1m` or so, then probably we can make use of `duration`. What do you think?